modules/core/src/main/scala/com.snowplowanalytics.snowplow.lakes/processing/SparkUtils.scala

```diff
@@ -70,7 +70,11 @@ private[processing] object SparkUtils {
     for {
       _ <- Logger[F].debug(s"Initializing local DataFrame with name $viewName")
       _ <- Sync[F].blocking {
-             spark.emptyDataFrame.createTempView(viewName)
+             try {
+               spark.sparkContext.setLocalProperty("spark.scheduler.pool", "pool1")
```

Comment:
If I understand correctly, `pool1` has higher priority than default pool. Are we using it for the light-weight jobs ?

Reply:
Yes exactly.  The lightweight jobs only.  I want to make sure they don't get stuck in a queue weighting for the slower jobs.